interface/resources/qml/controlsUit/ToolTip.qml

```diff
@@ -20,7 +22,8 @@ Item {
 
         width: toolTipText.width + 4
         height: toolTipText.height + 4
-        opacity: (toolTip != "" && showToolTip) ? 1 : 0
+        opacity: (toolTip != "" && showToolTip) ? 1 : 0	// Doesn't work.
+        //opacity: 1					// Works.
```

Comment:
Why doesn't it work?

Reply:
Not sure if I am looking at this right, but it `1 : 0` a division by zero?

Reply:
Julian: No, '?' is a ternary operator. https://en.wikipedia.org/wiki/%3F:#C++

Dale: I have no idea. That's why I couldn't get it to work. See my previous comment on this P.R. I tried several things to isolate the issue but had to give up before figuring it out. Sorry.